site/source/tutorials/9001_dojo1_dijits/index.md

```diff
@@ -0,0 +1,187 @@
+
+{% section %}
+
+## Updating project configuration
+
+{% task 'Update the `dojo build` configuration.' %}
+
+Dojo 1 and Dijits at runtime are considered _external_ dependencies for the Dojo 2 build system.  We need to supply information to the `dojo build` CLI command to include our external dependencies as part of our build.  The way we provide additional configuration information to the Dojo 2 CLI is via a `.dojorc` file in the root of our project.  The file is a JSON file, where configuration can be provided to each command.
+
+{% instruction 'Create a `.dojorc` with the following content:' %}
+
+{% include_codefile 'demo/finished/biz-e-corp/.dojorc' %}
+
+In this case we have declared four external dependencies which will be copied as an external dependency during our build.  In addition some of those dependencies will be injected (loaded) into our bundle.  The first one is a configuration file for the Dojo 1 loader (which we haven't created yet).  The second one includes the `dojo` package and loads the Dojo 1 AMD loader which will then subsequently load the other Dojo 1 modules that are requested.  The final dependency is a modern flat theme for Dojo 1 Dijits.  We are injecting the _root_ CSS file into our application bundle as well, which will ensure the CSS is loaded an available at run time.
+
+{% task 'Update the `tsconfig.json` configuration.' %}
+
+In order for us to integrate the TypeScript typings, we need to adjust two sections of our `tsconfig.json`.
+
+{% instruction 'Add `scripthost` to `lib` section of `tsconfig.json`:' %}
+
+{% include_codefile 'demo/finished/biz-e-corp/tsconfig.json' lines:6-14 %}
+
+Some of the Dojo 1 typings reference typings from this library.
+
+{% instruction 'Add the Dijit module typings to `include` section of `tsconfig.json`:' %}
+
+{% include_codefile 'demo/finished/biz-e-corp/tsconfig.json' lines:27-31 %}
+
+By adding the modules to the scope of the TypeScript compiler, we should be able to have typing information when attempting to import various Dijit modules.
+
+{% section %}
+
+## Configuring Dojo 1
+
+{% task 'Create loader configuration.' %}
+
+{% aside 'Information' %}
+The Dojo 1 loader configuration can be complex.  There is this Dojo 1 tutorial <em>[Configuring Dojo with dojoConfig](https://dojotoolkit.org/documentation/tutorials/1.10/dojo_config/index.html)</em> that explains how to configure Dojo 1.
+{% endaside %}
+
+In the previous step, we need to include a configuration for our Dojo 1 loader.  For this tutorial, we only need to ensure that the Dojo 1 loader works in an asynchronous fashion, because the Dojo 1 loader will default to synchronous operation, which will perform slower and cause issues with our application.  The file could be named anything, but in this tutorial, we are naming it `/src/dojo1.cfg.js`.
```

Comment:
which will perform slower and cause issues with our application -> which would inhibit performance and cause issues with our Dojo 2 application